### PR TITLE
Added README , SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING 

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added README , SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING 

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,27 @@
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a00c3917c3514e9b94bb8a181ba6db2e)](https://app.codacy.com/gh/Seagate/cortx-fs?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-fs&utm_campaign=Badge_Grade)
 
+# CORTX-FS
```
Comment (sachinpunadikar) [inline README.md:3]: keep it CORTXFS

File: README.md [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
 
-# CORTXFS
+The CORTX File System (FS) framework on top of the cloud object storage is the entry point for file access protocols (like NFS, CIFS). The CORTX-FS makes use of the NSAL and DSAL libraries to provide the file system framework. The [NSAL library](https://github.com/Seagate/cortx-nsal) provides namespace for the file system framework and the [DSAL library](https://github.com/Seagate/cortx-dsal) provide data storage functionality for the file system framework.
```
Comment (sachinpunadikar) [inline README.md:5]: CORTX-FS  -> CORTXFS

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Make sure that we are calling it CORTXFS

File: README.md (hunk 1 above)
Comment (Saumya-Sunder) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Saumya-Sunder) [reply README.md]: Done

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by s-arya — APPROVED** [review]
LGTM

**Pull request merged**